src/main/java/com/iexec/worker/compute/ComputeController.java

```diff
@@ -50,23 +50,24 @@ public ComputeController(final ComputeExitCauseService computeStageExitService,
      * @deprecated Use {@link #sendExitCausesForGivenComputeStage(String, ComputeStage, String, List)}
      * for bulk exit cause reporting instead
      */
-    @Deprecated(forRemoval = true) // TODO: Add version when releasing next one
+    @Deprecated(forRemoval = true)
     @PostMapping("/compute/{stage}/{chainTaskId}/exit")
     public ResponseEntity<Void> sendExitCauseForGivenComputeStage(
             @RequestHeader("Authorization") String authorization,
             @PathVariable ComputeStage stage,
             @PathVariable String chainTaskId,
             @RequestBody ExitMessage exitMessage) {
-        List<ReplicateStatusCause> causes = exitMessage != null && exitMessage.cause() != null ? List.of(exitMessage.cause()) : List.of();
-        return sendExitCausesForGivenComputeStage(authorization, stage, chainTaskId, causes);
+        List<WorkflowError> errors = exitMessage != null && exitMessage.cause() != null ?
```

Comment:
For the fun of it, a functional variant seems feasible here
```suggestion
        final List<WorkflowError> errors = Optional.ofNullable(exitMessage)
                .map(ExitMessage::cause)
                .map(WorkflowError::new)
                .stream().toList();
```

Reply:
Resolved in [79719d8](https://github.com/iExecBlockchainComputing/iexec-worker/pull/662/commits/79719d8a7c1f50dfc797f0f6c7424e2750c1a5ad)